src/alvin0319/AmongUs/game/Game.php

```diff
@@ -202,7 +202,7 @@ public function setMapItem(FilledMap $map) : void{
 	public function addPlayer(Player $player) : void{
 		if(!$this->hasPlayer($player)){
 			$this->players[] = $player->getName();
-			$this->broadcastMessage("Player " . $player->getName() . " has joined the game.");
+			$this->broadcastMessage(AmongUs::$prefix . "§e" . $player->getName() . " has joined the game.");
```

Comment:
Game::broadcastMessage has its own prefix, do not use AmongUs::$prefix

Reply:
k